myapp/spec/rails_helper.rb

```diff
@@ -73,6 +73,17 @@
   Capybara.server_port = 3000
   Capybara.app_host = "https://#{Capybara.server_host}:#{Capybara.server_port}"
 end
+
+if Bullet.enable?
+  config.before(:each) do
+    Bullet.start_request
+  end
+
+  config.after(:each) do
+    Bullet.perform_out_of_channel_notifications if Bullet.notification?
+    Bullet.end_request
+  end
+end  
```

Comment:
Checks for inconsistent indentation.
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/IndentationConsistency

Reply:
I have changed the format of this please check it once. 🙇‍♀️